Tests/LibJS/test-invalid-unicode-js.cpp

```diff
@@ -7,54 +7,130 @@
 #include <LibJS/Parser.h>
 #include <LibTest/TestCase.h>
 
-TEST_CASE(invalid_unicode_only)
+static bool produces_eofs(JS::Lexer& lexer)
```

Comment:
I don't think 'end of file' can have a plural, by definition :^)

```suggestion
static bool produces_eof_token(JS::Lexer& lexer)
```